USAGE_GUIDE.md

```diff
@@ -178,6 +178,35 @@ const iteratorId = resp.stack[0].id as string;
 const resp2 = await wcSdk.traverseIterator(sessionId, iteratorId, 10)
 ```
 
+## Encrypt and Decrypt data
+
+```ts
+// 1) encrypt data using the public key of the recipient, so only the recipient can decrypt it with his private key
+// this method receives an array of public keys, so you can encrypt the data for multiple recipients
+// and it returns an array of encrypted messages, one for each recipient public key
+const encryptedMessages = wcSdk.encrypt("Data to be encrypted", [recipientPublicKey])
+
+// 2) select which encrypted message you want to use
+const encryptedMessage = encryptedMessages[0]
+
+// 3) on the other side, the recipient can decrypt the data using his private key
+const messageDecrypted = wcSdk.decrypt(encryptedMessage)
+```
+On the example above, we used the `decrypt` with a single encrypted message, which is faster, but you can use
+`decryptFromArray` to pass multiple encrypted messages, and the recipient will try to decrypt until it finds the correct
+one.
+```ts
+// 1) encrypt data using the public key of many recipients
+const encryptedMessages = wcSdk.encrypt("Data to be encrypted", [
+    recipientPublicKey1, recipientPublicKey2, recipientPublicKey3
```

Comment:
>    recipientPublicKey1, recipientPublicKey2, recipientPublicKey3

I think it might be better to rename these variables, otherwise, it might seem that the same recipient has multiple public keys